pkg/sessions/utils/utils.go

```diff
@@ -2,19 +2,77 @@ package utils
 
 import (
 	"encoding/base64"
+	"errors"
 
 	"github.com/oauth2-proxy/oauth2-proxy/pkg/apis/sessions"
 	"github.com/oauth2-proxy/oauth2-proxy/pkg/encryption"
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/sessions/envelope"
 )
 
 // CookieForSession serializes a session state for storage in a cookie
```

Comment:
Should this `utils` area by refactored and just move this logic directly into the cookie_session? That seems to be the only place that uses these.

Reply:
Yes, I think that makes sense